core/src/main/java/com/datastrato/gravitino/meta/CatalogEntity.java

```diff
@@ -123,6 +123,40 @@ public CatalogInfo toCatalogInfo() {
     return new CatalogInfo(id, name, type, provider, comment, properties, auditInfo, namespace);
   }
 
+  public Catalog toBasicCatalog() {
```

Comment:
Maybe to `toCatalogInfo`, it is not `BasicCatalog` actually.

Reply:
Also I was thinking that can we just slightly change the `CatalogInfo` to be inherited from `catalog`, so maybe we can use `toCatalogInfo` directly.

Reply:
fixed